Sources/ManageServer.php

```diff
@@ -323,7 +323,7 @@ function AlignURLsWithSSLSetting($new_force_ssl = 0)
 			$smcFunc['db_query']('', '
 				UPDATE {db_prefix}themes
 				   SET value = {string:theme_val}
-				 WHERE variable = {string:theme_var}
+				WHERE variable = {string:theme_var}
```

Comment:
There's some weird indentation happening in this whole query, too. It should probably all be fixed, not just this one line.